schema.graphql

```diff
@@ -1852,6 +1853,7 @@ type Claim @entity {
   jurisdiction: String
   scope: Scope
   cddId: String
+  customClaimTypeId: BigInt
```

Comment:
How does this get populated? Doesn't the `Claim.create` call need to be updated to record this?

https://github.com/PolymathNetwork/polymesh-subquery/blob/3398d5eb462a300960752d8bbb3f53625e44b5fd/src/mappings/entities/mapClaim.ts#L101